shell/assets/translations/en-us.yaml

```diff
@@ -671,7 +671,9 @@ backupRestoreOperator:
     bucketName: Bucket Name
     credentialSecretName: Credential Secret
     endpoint: Endpoint
-    endpointCA: Endpoint CA
+    endpointCA:
+      label: Endpoint CA
+      prompt: Endpoint CA must be Base64 encoded
```

Comment:
Might have got my directions wrong, but now aren't we expecting it un-encoded... and we'll then encode it with `btoa` (which I think was the requirement from https://github.com/rancher/dashboard/issues/4903#issuecomment-1028149985)?